src/server/session.ts

```diff
@@ -677,13 +693,10 @@ namespace ts.server {
             return occurrences.map(occurrence => {
                 const { fileName, isWriteAccess, textSpan, isInString } = occurrence;
                 const scriptInfo = project.getScriptInfo(fileName);
-                const start = scriptInfo.positionToLineOffset(textSpan.start);
-                const end = scriptInfo.positionToLineOffset(textSpanEnd(textSpan));
                 const result: protocol.OccurrencesResponseItem = {
-                    start,
-                    end,
                     file: fileName,
                     isWriteAccess,
+                    ...this.toLocationTextSpan(textSpan, scriptInfo)

```

Comment:
this will cause an extra object allocation for every occurrence. we use this on every hover, and do not want to increase the amount of garbage we create.

Reply:
Rollback to be inline.